src/store/index.js

```diff
@@ -71,6 +74,11 @@ export default new Vuex.Store({
             Object.assign(state, JSON.parse(localStorage.getItem('store')))
           )
         }
-     }
+      }
+    },
+    actions: {
+        deleteCollection(context, id){
```

Comment:
you can use ES6 argument destructuring 
```
deleteCollection({ commit }, id) {
  commit('deleteCollection', id)
}
```
it is not so helpful in this case, but sticking to this technique would be helpful in more complicated instances